test/fixtures/settings.yml

```diff
@@ -1,13 +1,10 @@
 ---
```

Comment:
Can we merge https://github.com/theforeman/foreman/pull/9530 first? Then we simply get rid of the file altogether.

Reply:
Sure :shrug: I don't think the order matters much here tho, if #9530 gets in first, I'm happy to rebase, but I don't think it should be a blocker :)